olp-cpp-sdk-core/include/olp/core/generated/parser/JsonParser.h

```diff
@@ -60,5 +62,16 @@ inline T parse(std::stringstream& json_stream) {
   return parse<T>(json_stream, res);
 }
 
+template <typename T>
+inline T parse(const std::shared_ptr<std::vector<unsigned char>>& json_bytes) {
+  rapidjson::Document doc;
+  doc.Parse(reinterpret_cast<char*>(json_bytes->data()), json_bytes->size());
+  T result{};
+  if (doc.IsObject() || doc.IsArray()) {
+    from_json(doc, result);
+  }
+  return result;
+}
```

Comment:
We should provide a variant with a parsing result, or else a default value is returned.

Reply:
This one is similar to other `parse` functions, the only difference is the source.